orb-supervisor/tests/it/helpers.rs

```diff
@@ -190,8 +189,8 @@ pub async fn start_interfaces(
     dbus_instances: &DbusInstances,
 ) -> zbus::Result<zbus::Connection> {
     let conn = zbus::ConnectionBuilder::address(dbus_instances.system.address())?
-        .name(zbus_systemd::systemd1::ManagerProxy::DESTINATION)?
-        .serve_at(zbus_systemd::systemd1::ManagerProxy::PATH, Manager)?
+        .name(zbus_systemd::systemd1::ManagerProxy::DESTINATION.unwrap())?
```

Comment:
if we're about to panic here, would you consider `expect()` with super short explanation? I would have no clue how to fix/debug the panic here without going deeper as I can't find the DESTINATION and PATH values in our code nor the `zbus_systemd` crate.

Reply:
Ok actually no need for more info here - can be figured out from the code